src/Cdn/Cdn.Autorest/README.md

```diff
@@ -51,9 +51,12 @@ identity-correction-for-post: true
 resourcegroup-append: true
 nested-object-to-string: true
 
+disable-transform-identity-type: true
+flatten-userassignedidentity: false
+
 # For new modules, please avoid setting 3.x using the use-extension method and instead, use 4.x as the default option
 use-extension:
-  "@autorest/powershell": "3.x"
+  "@autorest/powershell": "4.x"
```

Comment:
4.x is the by default value. If you plan to migrate to 4, then can remove the `use-extension` attribute.

Reply:
Removed.